main.go

```diff
@@ -1,7 +1,51 @@
 package main
 
-import "fmt"
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+)
+
+type Domicile struct {
+	country   string `json:"country"`
+	is_remote bool   `json:"is_remote"`
+}
+
+type Employee struct {
+	name            string   `json:"name"`
+	entity          string   `json:"entity"`
+	employee_number int      `json:"employee_number"`
+	salary          float64  `json:"salary"`
+	domicile        Domicile `json:"domicile"`
```

Comment:
OOT - have you tried what if you forgot defining the json postfix on here?

Reply:
I've tried but it's on assignment 1, i forgot about the json naming then the class doesn't recognized the json attributes inside class